recipe/meta.yaml

```
@@ -31,14 +33,18 @@ test:
   commands:
     - pip check
     - pip install -e ".[test]"
-    - python -m pytest -vv
+    - python -m pytest -k "not (test_run or test_normalize_cmd)" -vv  # [win]
+    - python -m pytest -vv  # [not win]
   requires:
     - pip
+    - cryptography =39.0.1
```

```suggestion
    - pytest
    - pytest-mock
    - hatch
    - tomli
    - twine
```
https://github.com/jupyterlab/hatch-jupyter-builder/blob/v0.8.3/pyproject.toml#L35

---

`twine` is required for publishing on PyPi so we do not need it for testing